builtin/v10/migration/top.go

```diff
@@ -221,6 +243,61 @@ func MigrateStateTree(ctx context.Context, store cbor.IpldStore, newManifestCID
 		return cid.Undef, xerrors.Errorf("migration group error: %w", err)
 	}
 
+	// Make the empty state
+
+	emptyObj, err := makeEmptyState(store)
+	if err != nil {
+		return cid.Undef, xerrors.Errorf("failed to make empty obj: %w", err)
+	}
+
+	// Create the EAM Actor
+
+	eamAct, err := CreateEAMActor(&newManifest, emptyObj)
+	if err != nil {
+		return cid.Undef, xerrors.Errorf("failed to create the EAM Actor: %w", err)
+	}
+
+	if err := actorsOut.SetActorV5(builtin.EthereumAddressManagerActorAddr, eamAct); err != nil {
+		return cid.Undef, xerrors.Errorf("failed to set EAM Actor: %w", err)
+	}
+
+	// Create the EthZeroAddress as an EthAccount
```

Comment:
I'm not seeing us _create_ it. Are we just checking to see if it exists?

Reply:
WDYM? There's basically 2 steps to "creating" it:

- Assigning it an ID in the init actor (happens as part of the init actor migration)
- Adding that ID to the state tree as an EthAccount, which happens ~25 lines after this comment.

We're _not_ assigning it a robust address, but that seemed okay to me?

Reply:
> Assigning it an ID in the init actor (happens as part of the init actor migration)

Ah, this is what I was looking for. You're right, we're creating the actor here, we've already assigned the address.